src/test/testJsonValidator.js

```diff
@@ -0,0 +1,26 @@
+import { errorTypes } from '../constants.js';
+import { jsonIsValid } from '../jsonValidator.js'
+
+export const testJsonValidator = () => {
+  const errors=[];
+
```

Comment:
If in the future we need to add more test cases, it would be easier to maintain this code if we implement a more dynamic approach to the test cases.

I would recommend taking the following approach:
1.  Define a `testCases` array that contains objects with the following structure: {jsonString: "...", case: "..." } where jsonString is the input to the function and case is the string that identifies the case to test.
2. Loop through the testCases array and call the functions needed.

This way, when we need to add a new test case we just add it to the array and not add new code for it.